aggregate.go

```diff
@@ -78,19 +84,32 @@ type spanGroup struct {
 	Slowest []spanGroupSlowest // N-slowest traces in the group.
 }
 
+func (s spanGroup) Len() int      { return len(s.Slowest) }
+func (s spanGroup) Swap(i, j int) { s.Slowest[i], s.Slowest[j] = s.Slowest[j], s.Slowest[i] }
+func (s spanGroup) Less(i, j int) bool {
+	a := s.Slowest[i]
+	b := s.Slowest[j]
+	return a.End.Sub(a.Start) > b.End.Sub(b.Start)
```

Comment:
In my mind I think of `sort.Reverse` as reversing the normal order of something -- given that I would expect `Slowest` to be sorted in order of slowest-to-fastest I would think `sort.Reverse` meant sorting in order of fastest-to-slowest.

> That'd make a good comment to use.

Good point, I'll add that.